### ISSUE TITLE
2.0.7 : Can't translate FPT text on checkout/cart page

### ISSUE THREAD

**Issue by kanduvisla** [magento/magento2#5477]
**Preconditions:**

Magento 2.4-develop
Php v 7.3

**Steps to reproduce**

1. Enable FPT in admin; (Go to Admin->Stores->Configuration->Sales->Tax->Fixed Product Taxes);
2. Change translate to FPT `FPT,FPT(translate)` for example (app/code/Magento/Weee/i18n/en_US.csv);
3. Create simple product with FPT attribute and set value for it;
![Screenshot from 2020-04-30 12-05-32](https://user-images.githubusercontent.com/51679138/80692950-fec44280-8ada-11ea-8df5-59ec051ec8d0.png)
4. Go to Storefront;
5. Add created before the product to Cart;
6. Go to Shopping Cart;

:heavy_check_mark: The FPT has been translated
![screenshot](https://user-images.githubusercontent.com/51679138/80693332-8447f280-8adb-11ea-89a9-c7b664cf4aeb.png)
7. Proceed to Checkout;
8. Fill all required fields and go to **Review&Payment** step;

**Actual Result:**

:heavy_multiplication_x: The FPT was not translated
![screenshot_1](https://user-images.githubusercontent.com/51679138/80693809-3c759b00-8adc-11ea-9ebd-b68d25163b0e.png)

**Expected Result:** 

:heavy_check_mark:  The FPT must be translated

I try to translate the text 'FPT' in the checkout page but nothing seems to work:
I try with a language pack in `app/i18n/Vendor/nl_nl/nl_NL.csv` (the rest of the translations of this pack work so it is loaded):
```
"FPT","Verwijderingsbijdrage",module,Magento_Weee
"Magento_Weee::FPT","Verwijderingsbijdrage",module,Magento_Weee
```
I try with the file `app/design/frontend/Vendor/Theme/i18n/nl_NL.csv`:
```
"FPT","Verwijderingsbijdrage"
"Magento_Weee::FPT","Verwijderingsbijdrage"
```
I try with the file `app/code/Magento/Weee/i18n/nl_NL.csv`:
```
FPT,Verwijderingsbijdrage
```
I try by adding an entry to the 'translation'-table in the database (even had to manually check the crc_string):
```
key_id  string  store_id    translate               locale  crc_string
2       FPT     0           Verwijderingsbijdrage   nl_NL   1575739609
```
All with no success.
I'm running out of options. Is there any spot I missed where I need to put my translation? Can anyone confirm / try if the text 'FPT' in the checkout/cart page is translatable?

**Comment by veloraven** [comment]
@kanduvisla thank you for your report.
Internal ticket MAGETWO-56471 was created for this issue.

**Comment by veloraven** [comment]
Has a fix in #5732 
Duplicate #5731

**Comment by magento-engcom-team** [comment]
@kanduvisla, thank you for your report.
The issue is already fixed in 2.2.0

**Comment by hostep** [comment]
Re-opening because the FPT label is not translatable yet on the checkout page (second step).

The fix was only done to [the `checkout_cart_index.xml` file](https://github.com/magento/magento2/blob/a73532a0a7bf5cdcfe298f5cfb6f7ce8405b977c/app/code/Magento/Weee/view/frontend/layout/checkout_cart_index.xml#L20) in https://github.com/magento/magento2/pull/9204, but was forgotten to be done to [the `checkout_index_index.xml` file](https://github.com/magento/magento2/blob/a73532a0a7bf5cdcfe298f5cfb6f7ce8405b977c/app/code/Magento/Weee/view/frontend/layout/checkout_index_index.xml#L26).

**Comment by engcom-Alfa** [comment]
The issue is reproducible on a fresh 2.4-develop instance.

**Manual testing scenario:**

1. Enable FPT in admin; (Go to Admin->Stores->Configuration->Sales->Tax->Fixed Product Taxes);
2. Change translate to FPT `FPT,FPT(translate)` for example (app/code/Magento/Weee/i18n/en_US.csv);
3. Create simple product with FPT attribute and set value for it;
![Screenshot from 2020-04-30 12-05-32](https://user-images.githubusercontent.com/51679138/80692950-fec44280-8ada-11ea-8df5-59ec051ec8d0.png)
4. Go to Storefront;
5. Add created before the product to Cart;
6. Go to Shopping Cart;

:heavy_check_mark: The FPT has been translated
![screenshot](https://user-images.githubusercontent.com/51679138/80693332-8447f280-8adb-11ea-89a9-c7b664cf4aeb.png)
7. Proceed to Checkout;
8. Fill all required fields and go to **Review&Payment** step;

**Actual Result:**

:heavy_multiplication_x: The FPT was not translated
![screenshot_1](https://user-images.githubusercontent.com/51679138/80693809-3c759b00-8adc-11ea-9ebd-b68d25163b0e.png)

**Expected Result:** 
:heavy_check_mark:  The FPT must be translated

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-33870` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by slavvka** [comment]
Hi @kanduvisla. Thank you for your report. 
The issue has been fixed in magento/magento2#28108 by @engcom-Charlie in 2.4-develop branch
Related commit(s):
 - [f1e558cd2cf13a574b4853fa478c40fd00fb4cbb](https://github.com/magento/magento2/commit/f1e558cd2cf13a574b4853fa478c40fd00fb4cbb)
 - [f80fb3035e21b9978a4ad814e122188d4702a69e](https://github.com/magento/magento2/commit/f80fb3035e21b9978a4ad814e122188d4702a69e)

The fix will be available with the upcoming 2.4.0 release.
